plugins/wfp/common.py

```diff
@@ -325,9 +326,12 @@ def journey_Formatter(self, visit, anthropometric_visit_form, followup_forms, cu
             current_journey["weight_difference"] = visit.get("weight_difference", None)
             current_journey["exit_type"] = self.exit_type(visit)
 
-        if index > 0:
+        if index > 3:
             index = index - 1
-        exit = self.exit_by_defaulter(visits, visits[index], followup_forms)
+            exit = self.exit_by_defaulter(visits, visits[index], followup_forms)
+        else:
+            exit = self.exit_by_defaulter(visits, visit, followup_forms)
+
```

Comment:
Not sure I understand these changes :thinking: can you explain the change to index `3`? Why the magic number?

Reply:
When it's index 3 it means It's a first visits. There is admission visit which considered as visit 0 with 3 forms: registration+Anthropometric admission+Medical admission+Assistance Admission) and when it's index 4 it means it's starting the visit 1(Followup visit) after admission visit(visit 0) . 

In some case, there are beneficiaries that never return after first followup visit(visit 1), and we need to get the number of days in previous visits form which is the assistance admission form(with index 3) in order to calculate the 2 next visits dates and flag **as defaulter the beneficiary** if there is no data in the 2 calculated next visits date. When the index is less or equal to 3, it means we still in the admission visit.

Reply:
For example, the following beneficiary missed 2 consecutive visits: https://qa.coda.go.wfp.org/dashboard/entities/details/accountId/1/entityId/254: 
1. Next visit date on: 25/07/2023
2. Second visit date on: 08/08/2023

We need to calculate the 2 next visits dates based on the number of days defined in [assistance admission form](https://qa.coda.go.wfp.org/dashboard/forms/submission/accountId/1/instanceId/1759) **and flag the case as defaulter**.

![Coda-2-Admission-Assistance-6_4_db316cad-434b-4082-9751-39dae654e647_2023-07-11_16-54-30](https://github.com/user-attachments/assets/4b80d3d4-3bfc-4d1d-9e67-6cbe36a29b5a)

Reply:
@bramj , I changed a bit on: https://github.com/BLSQ/iaso/pull/1556/files#diff-adf0bcaae996ccef6e87dcac984ce60a2d7419bba436eeca75ea15fb33153d20R335

It checks if it's followup visit form. This more safer and it also help to know if it was a follow up visit which could be visit 1 or 2 or 3, etc.....

Reply:
Nice :+1: looks safer indeed!